boards/arm/nrf9160_innblue22/Kconfig.defconfig

```diff
@@ -0,0 +1,57 @@
+# nRF9160 innblue V2.2 board configuration
+
+# Copyright (c) 2020 innblue
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF9160_INNBLUE22 || BOARD_NRF9160_INNBLUE22NS
+
+config BOARD
+	default "nrf9160_innblue22"
+
+config ADC_0
+	default y
```

Comment:
That ADC error is likely because this is default y, but you'd need to look into the sanitycheck configuration for your board more to be sure.

https://docs.zephyrproject.org/latest/guides/test/sanitycheck.html

Reply:
ov! got your point -- thanks...

that is the only place for ADC and SPI... it is not used, so safe to remove.

Let me check it again.

Reply:
> that is the only place for ADC and SPI... it is not used, so safe to remove.

Are you sure it is OK to remove?

Just turning off ADC and SPI support to make the complaining test go away is not necessarily what should be done. The board porting guide has the requirements on what peripherals a board should be enabling:

https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#general-recommendations

In particular:

> Configure components that enable the use of these pins, such as configuring an SPI instance to use the usual Arduino SPI pins.

Please make sure to follow the guidelines, updating the tests as needed to pass, instead of just silencing test errors.

Reply:
> > Configure components that enable the use of these pins, such as configuring an SPI instance to use the usual Arduino SPI pins.
> 
> Please make sure to follow the guidelines, updating the tests as needed to pass, instead of just silencing test errors.

I'm not sure whether this has been explicitly addressed. It needs to be.

Reply:
@mbolivar , The innblue 2.2 board doesn't have support for ADC / SPI. As we used the nRF9160 dev kit as a baseline for our board, these definitions renamed in place. Now we've decided to remove them altogether. 

In short, we don't want to fix the application build errors for our board because it doesn't have supported for the mentioned peripherals.